Makefile

```
@@ -1,3 +1,9 @@
 tests:
 
 .PHONY: tests
+
+build:
+	docker build -t laysan-albatross-sexual-dimorphism .

```

Por favor por la versión larga de la instrucción:
```suggestion
	docker build --tag laysan-albatross-sexual-dimorphism .

```